### ISSUE TITLE
Update or find alternatives to obsolete addons

### ISSUE THREAD

**Issue by Nutser** [mazen160/Firefox-Security-Toolkit#4]
The following list of addons no more supported in the latest builds of Firefox or not exists no more.

- PassiveRecon
- Tamper Data
- Multi Fox
- Disable WebRTC
- Crypto Fox

Please, find alternatives for them.

**Comment by mazen160** [comment]
Hi @Nutser,

Thank you for reporting the issue and listing the currently unsupported add-ons.

I will be testing it and release a new update.

Thanks!,
Mazin

**Comment by mazen160** [comment]
Hi @Nutser,

I have released a new update of Firefox-Security-Toolkit.

I have removed Multi Fox add-on per your report.

Regarding the following add-ons:
* PassiveRecon
* Tamper Data
* Crypto Fox

Although the add-ons are not fully-updated to meet Mozilla's standards, there are currently working normally in the latest version of Firefox. Therefore, I can not remove them. However, the add-ons are working normally.

Regarding, Disable WebRTC, it seems to be updated as of today.
 

Thanks again for reporting!

Best,
Mazin
